flake8_dcos_lint/setup.py

```diff
@@ -18,7 +18,7 @@
     install_requires=[
         'pycodestyle==2.2.0',
         'flake8',
-        'flake8-import-order==0.9.2',
```

Comment:
I did this while trying to fix some other naming errors. Any objections to leaving it?